front-end/src/screens/OnChain/index.tsx

```diff
@@ -21,12 +22,16 @@ const OnchainPostsContainer = ({ className } : {className?: string}) => {
 				<Grid.Column mobile={16} tablet={16} computer={10}>
 					<h3>Referenda</h3>
 					<ReferendaContainer className='referendaContainer' limit={5} />
+					<Link className='onchain-link' to='referenda'><Icon name='arrow right'/>See all referenda</Link>
```

Comment:
nit, this classname name is misleading. on-chain links are the name of a table in hasura to link the discussion with onchain data. I'd suggest something like "full-listing-link"